Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: format
 format:
-	venv/bin/black -t py35 -t py36 -t py37 -t py38 -t py39 -t py310 -t py311 -t py312 .
+	venv/bin/ruff format --target-version py37 .
```

Comment:
Python 3.7 is the earliest version we can target, per
https://docs.astral.sh/ruff/settings/#target-version

Reply:
I should have mentioned, by the way, that unlike `black` it doesn't want you to specify every version you support - just the lowest.